src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -212,23 +370,33 @@
  * Example of allowed public immutable fields:
```

Comment:
Please format all xml configs to wrap long lines at name of attribute.
Indentation should be +2 on wrapping.

Please do for all xml configs

Reply:
Example:
```
<module name="VisibilityModifier">
  <property name="allowPublicImmutableFields" value="true"/>
  <property name="immutableClassCanonicalNames"
     value="com.google.common.collect.ImmutableSet"/>
</module>
```

Reply:
All is done.
Please, if there are other changes can you tell me all at once? 
I worked and solved other issues in my local, but the last time i tried to send them in another PR everything was crashed, so i'm waiting this to be merged.
In this way i will not have any chance in GSoC.
thanks

Reply:
@0xbakry review is an iterative process, just like learning and writing code :)  git is a critical and necessary skill, take the time to learn it.

> All is done

Reply to each individual item with "done" or disagreement, it makes review easier and makes sure that we do not miss anything.

Reply:
@nrmancuso yeah defiantly after all of that i'll put more effort on it, you all helped me a lot, thanks (:

> Reply to each individual item with "done" or disagreement, it makes review easier and makes sure that we do not miss anything.

Great, will do this now.